beacon_node/eth2_libp2p/src/discovery/mod.rs

```diff
@@ -53,7 +53,8 @@ pub const TARGET_SUBNET_PEERS: usize = config::MESH_N_LOW;
 const TARGET_PEERS_FOR_GROUPED_QUERY: usize = 6;
 /// Number of times to attempt a discovery request.
 const MAX_DISCOVERY_RETRY: usize = 3;
-/// The maximum number of concurrent discovery queries.
+/// The maximum number of concurrent subnet discovery queries.
+/// Note: we always allow a single FindPeers query.
 const MAX_CONCURRENT_QUERIES: usize = 2;
```

Comment:
Maybe in the note we say that this means we will be running at most MAX_CONCURRENT_QUERIES + 1 to make it explicit.